### ISSUE TITLE
chest tracking line goes invisible with slidurs shaders

### ISSUE THREAD

**Issue by iceyrazor** [AranaiRa/ArcaneArchives#124]
i dont really expect this too be fixed. but if you could. that would be great.

basically. when i have slidurs vibrant shaders extreme. the line that follows items in a chest goes invisible.

theres not much else i can add to this

what might be a very possible fix. is a option to make the line have a opacity of 100% and emit little light. or not emit light

**Comment by noobanidus** [comment]
Unfortunately, it's really difficult to reconcile shaders with this. I'm just using the standard "lines" mode of OpenGL as the render type. I can look at setting the lightmap to be fullbright, but I'm not 100% sure if that would help. Likewise, getting OptiFine for Shaders to work inside of a development environment is next to impossible :(

**Comment by iceyrazor** [comment]
> Unfortunately, it's really difficult to reconcile shaders with this. I'm just using the standard "lines" mode of OpenGL as the render type. I can look at setting the lightmap to be fullbright, but I'm not 100% sure if that would help. Likewise, getting OptiFine for Shaders to work inside of a development environment is next to impossible :(

yeah. worth a shot to ask though. i saw on another post that you planed on having it do a outline on the chest that you would see through walls.

i just now thought that if you were to also make it change its texture. then that would be more friendly with shaders. as it would have a different  texture
